.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*.{kt,kts}]
+
+[{engine-client-codegen/src/test/resources/**,engine-client-codegen/target/**,engine-maven-plugin/target/**,npl-maven-plugin/target/**}]
+ktlint_standard = disabled
+ktlint_experimental = disabled
+ktlint_custom-rule-set = disabled
```

Comment:
I think the reason we have `ktlint_standard = disabled` in platform is historical, I think. Not sure if it makes sense in new projects. Let's maybe go with the minimal example in https://pinterest.github.io/ktlint/0.49.1/rules/configuration-ktlint/?